.github/ISSUE_TEMPLATE.md

```diff
@@ -9,7 +9,7 @@ repo. https://github.com/nodejs/help
 Please fill in as much of the template below as you're able.
 
 Version: output of `node -v`
-Platform: output of `uname -a` (UNIX), or version and 32 or 64-bit (Windows)
+Platform: output of `uname -a` (UNIX), or output of `"$([Environment]::OSVersion | ForEach-Object VersionString) $(if ([Environment]::Is64BitOperatingSystem) { "x64" } else { "x86" })"` in Powershell console (Windows)
```

Comment:
`cmd /c ver` is probably more concise.

cc @nodejs/platform-windows

Reply:
It is for version, but it still doesn't have bitness